contracts/moneyflow/ether/WeiInfiniteFund.sol

```diff
@@ -0,0 +1,8 @@
+pragma solidity ^0.4.24;
+import "./WeiFund.sol";
+
+
+contract WeiInfiniteFund is WeiFund {
```

Comment:
Add comments for each new contract (in the std. format)

Reply:
What is the idea behind the WeiInfiniteFund?
Why don't we have a WeiInfiniteExpense for example?

Reply:
Because WeiInfiniteExpense will revert if amount is less then needed => revert always

Reply:
ok, let's rename that to WeiUncappedFund.

And add comment "This fund will keep receiving ANY amount of ETH forever"